packages/type-safe-api/scripts/type-safe-api/generators/python/templates/operationConfig.mustache

```diff
@@ -1,7 +1,7 @@
 from __future__ import annotations
 import urllib.parse
 import json
-from typing import Callable, Any, Dict, List, NamedTuple, TypeVar, Generic, Union, TypedDict, Protocol, Optional, Literal
+from typing import Callable, Any, Dict, List, NamedTuple, TypeVar, Generic, Union, TypedDict, Protocol, Optional, Literal, Annotated
```

Comment:
How are we using `Annotated`?

Reply:
The fields with descriptions use it, ie `{{paramName}}: {{{vendorExtensions.x-py-typing}}}` might become `name: Annotated[StrictStr, Field(..., description="This is a description")]` 😄